### ISSUE TITLE
useFirestoreDocData never loading data

### ISSUE THREAD

**Issue by gpsanant** [FirebaseExtended/reactfire#320]
### Version info

**React:** ^17.0.1

**Firebase:** ^8.2.1

**ReactFire:** ^3.0.0-rc.0

**Other (e.g. Node, browser, operating system) (if applicable):** Chrome

### Issue

Followed the tutorial within an express app. Set up react views and served the following file:
```js
import React, { Component, Suspense } from 'react';
import { render } from 'react-dom';

import 'firebase/database';
import { FirebaseAppProvider, useFirestoreDocData, useFirestore, SuspenseWithPerf, useFirestoreDocDataOnce } from 'reactfire';

const firebaseConfig = {
    // config
  };

function Burrito() {
  // easily access the Firestore library
  const burritoRef = useFirestore()
    .collection('tryreactfire')
    .doc('burrito');

  // subscribe to a document for realtime updates. just one line!
  const doc = useFirestoreDocData(burritoRef);
  check()
  // easily check the loading status
  if (doc.status === 'loading') {
    return <p>Fetching burrito flavor...</p>
  }

  return <p>The burrito is {doc.data.yummy ? 'good' : 'bad'}!</p>;
}

function App() {
  return (
    <FirebaseAppProvider firebaseConfig={firebaseConfig}>
      <h1>🌯</h1>
      <Burrito />
    </FirebaseAppProvider>
  );
}

async function check(){
    const burritoRef = useFirestore()
    .collection('tryreactfire')
    .doc('burrito')

    var x = await useFirestore()
    .collection('tryreactfire')
    .doc('burrito').get();
    console.log(x.data())
}

module.exports = App;
```

I expect the correct data to eventually load. The check function logs `{yummy: true}` like it should but when I continually log the return value of the useFirestoreDocData call, I get: 
```
{
  status: 'loading',
  hasEmitted: false,
  isComplete: false,
  data: undefined,
  error: undefined,
  firstValuePromise: Promise { <pending> }
}
{
  status: 'loading',
  hasEmitted: false,
  isComplete: false,
  data: undefined,
  error: undefined,
  firstValuePromise: Promise { undefined }
}
...
```

Any insight as to what I'm doing wrong? 
Thanks.

**Comment by jhuleatt** [comment]
Hey @gpsanant, it looks like the quickstart is out of date. Sorry about that. I just made some fixes to the readme that provides a working StackBlitz link and complete Firestore sample code. I'd recommend using that as a starting point as we work to make the quickstart better. [Here's a link](https://stackblitz.com/fork/reactfire-sample) to create a fork that you can edit.

**Comment by gpsanant** [comment]
Hey @jhuleatt, thanks for the reply. I put that code in my file but I unfortunately get the same result. The only difference between my code and the StackBlitz code is that you use react-dom to render the React component while I export the app component and render it using res.render in my Express app.js. Could this be messing something up?

**Comment by AbrarShahriar** [comment]
@jhuleatt  Your code actually worked for me...but the quickstart is just...*sighs
Please, update the instructions. That quickstart took 2 hours from my life😓

**Comment by jhuleatt** [comment]
Fixed in #326
